packages/webcomponents/src/components/business-forms/payment-provisioning/business-owners/business-owners-form-step-core.tsx

```diff
@@ -139,8 +139,11 @@ export class BusinessOwnersFormStepCore {
   render() {
     return (
       <div>
-        <legend>Owners</legend>
-        <hr />
+        <div class="d-flex align-items-center gap-2">
+          <legend class="mb-0">Owners</legend>
+          <form-control-tooltip helpText="For partnerships, LLCs or privately held corporations, the business is required to apply with all individuals with 25% or more ownership to the application. For charities and registered non-profits, the business is required to apply with 1 individual with substantial control over the entity; a board member or director." />
```

Comment:
I think there's a `;` after entity?

Reply:
Good catch! Thanks, I dont think that was correct. So I updated it as follows:
`the business is required to apply with 1 individual with substantial control over the entity, such as a board member or director.`